app/models/manageiq/providers/nuage/network_manager/cloud_subnet.rb

```diff
@@ -1,2 +1,60 @@
 class ManageIQ::Providers::Nuage::NetworkManager::CloudSubnet < ::CloudSubnet
+  supports :delete
+  supports :create
+
+  def delete_cloud_subnet_queue(userid)
+    task_opts = {
+      :action => "deleting Cloud Subnet for user #{userid}",
+      :userid => userid
+    }
+    queue_opts = {
+      :class_name  => self.class.name,
+      :method_name => 'raw_delete_cloud_subnet',
+      :instance_id => id,
+      :priority    => MiqQueue::HIGH_PRIORITY,
```

Comment:
Don't think this has to be high priority :)

Reply:
Copy-pasted from OpenStack 😇 It sounds reasonable to have user operations high priority because usually she waits to see immediate reaction, pressing F5 all the time, right?

Would you suggest normal priority anyway?

Reply:
Yeah that was incorrect when openstack did it as well, we should only use high priority when something _needs_ to jump ahead of everything else in the queue for things to work.